tasks/districts/FetchMumbaiWardsTask.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+import luigi
+
+from backend.data.extract_mumbai_wards_pdf import scrape_mumbai_pdf
+from tasks.districts.DownloadFileTask import DownloadFileTask
+
+
+class FetchMumbaiWardsTask(luigi.Task):
+    def requires(self):
+        return DownloadFileTask(
+            file_url='http://stopcoronavirus.mcgm.gov.in/assets/docs/Dashboard.pdf')
+
+    def run(self):
+        df = scrape_mumbai_pdf(self.input().path)
+
+        df.to_csv(self.output().path, index=False)
+
```

Comment:
just a note to myself for updating the phase 2 Ward PDF to Dropbox process: 
this is where you would suggest adding the method to write PDF to our drop box right?

and self.input().path points to the PDF file path on whatever machine the process is running on?

Reply:
correct. I've updated the code to have that as a comment in there as well.